testing/ios/IosUnitTests/IosUnitTests.xcodeproj/project.pbxproj

```diff
@@ -359,6 +359,7 @@
 				DEBUG_INFORMATION_FORMAT = dwarf;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
+				FLUTTER_ENGINE = ios_profile;
```

Comment:
Did you intend to change IosUnitTests?

Reply:
I looked into extending IosUnitTests at the beginning but ended up deciding against it. It was a while ago so I don't remember what exactly the issue(s) were but I think it was partly because we didn't need to pull in Flutter.xcframework and instead just the code surrounding displaylist/flow/etc. I think there may have been duplicate symbol issues but don't quote me on that.

Reply:
I'd just leave this out then.  Even if it should exist, it shoudn't be `profile`.

Reply:
What do you think it should be then? We use it to get the path to pull in the dylib. It needs to be either profile or release.

Reply:
This is the xcodeproj for the unit tests, not the benchmarks.  These should be running the debug engine.

Reply:
Oh! Sorry this should not have been in here, I'll delete.